R/mock-data.R

```diff
@@ -170,3 +170,56 @@ mockBulkeQTL <- function(n.genes = 500, seed=NULL){
 
     return(mock.eq)
 }
+
+
+#' Generate set of "empirical" mock data
+#' 
+#' Quick function to generate matching mock VCF, bulk expression, and eQTL data,
+#' useful for running splatPopEmpiricalMeans
+#' 
+#' @param n.genes Number of genes in mock eQTL data.
+#' @param n.snps Number of SNPs in mock vcf file.
+#' @param n.samples Number of samples in mock bulk data.
+#' @param chromosome Chromosome name
+#' @param chr.length Length of mock chromosome 
+#' @param seed Optional: seed for random seed
+#' 
+#' @return list(gff=mockGFF, vcf=mockVCF, means=mockMEANS, eqtl=mockEQTL)
+#'
+#' @examples
+#' empirical <- mockEmpiricalSet()
+#'
+#' @export
+#' 
+mockEmpiricalSet <- function(n.genes = 20, n.snps = 1000, n.samples = 10, 
+                    chromosome = 1,  chr.length = 2e6, seed=NULL){
```

Comment:
Can you please add spaces to `seed=NULL` (in the function as well)? Super picky I know but I try to keep things consistent if I can (I really should work out how to style things automatically).